fern/definition/medical/organization.yml

```diff
@@ -0,0 +1,94 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/fern-api/fern/main/fern.schema.json
+
+imports:
+  commons: ../commons.yml
+
+service:
+  base-path: /organization
+  auth: true
+  endpoints:
+    create:
+      display-name: Create Organization
+      docs: Registers your Organization in Metriport.
+      method: POST
+      path: ""
+      request: BaseOrganization
+      response: Organization
+      examples:
+        - request: 
+            name: Metriport Inc.
+            type: ambulatory
+            location: $commons.Address.Example1
+          response:
+            body: $Organization.Example1
+
+    get:
+      display-name: Get Organization
+      docs: Gets the Organization representing your legal corporate entity.
+      method: GET
+      path: ""
+      response: Organization
+      examples:
+        - response:
+            body: $Organization.Example1
+
+    update:
+      display-name: Update Organization
+      docs: Updates your Organization's details.
+      method: PUT
+      path: /{id}
+      path-parameters:
+        id:
+          type: string
+          docs: The ID of your organization.
+      request: BaseOrganization
```

Comment:
Great -- I defer to you on the naming here (you know better than me). I can make the changes but it might be faster if you went an updated, thoughts?